fix for last open items at https://github.com/OWASP/owasp-masvs/issues/203 (#1479)
OWASP/owasp-mstg: Document/0x06h-Testing-Platform-Interaction.md

- Identifying WebView usage
- Testing JavaScript configuration
- Testing for mixed content

> - Testing for webview URI manipulation

Reviewer: ```suggestion
- Testing for WebView URI manipulation
```

hasOnlySecureContent:  false
```

##### Testing for WebView URI Manipulation

> Make sure that the webview its URI cannot be manipulated by the user in order to load other types of resources than necessary for the functioning of the webview. This can be specifically dangerous the momen the webview resources come from the local file system, allowing the user to navigate to other resources within the application.

Reviewer: ```suggestion
Make sure that the WebView's URI cannot be manipulated by the user in order to load other types of resources than necessary for the functioning of the WebView. This can be specifically dangerous when the WebView's content is loaded from the local file system, allowing the user to navigate to other resources within the application.
```
